docs/src/components/Graph.stories.tsx

```diff
@@ -0,0 +1,78 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+import { Graph } from '@ui-kit-2022/components';
+import * as React from 'react';
+export default {
+  title: 'Components/Graph',
+  component: Graph,
+} as ComponentMeta<typeof Graph>;
+
+import Perlin from '../utils/perlin';
+const Template: ComponentStory<typeof Graph> = (props) => {
+  return <Graph {...props} />;
+};
+
+const perlin = new Perlin();
+let y = 0.008;
+
+const labels = Array(24 * 60)
+  .fill(0)
+  .map((_, i) => {
+    return ('0' + ~~(i / 60) + ': 0' + Math.round(60 * ((i / 60) % 1))).replace(
+      /\d(\d\d)/g,
+      '$1',
+    );
+  })
+  .filter((time) => {
+    const [hour, minute] = time.split(':');
+    return ['09', '10', '11', '12', '13', '14', '15', '16'].includes(hour);
+  });
+
+export const GraphStory = Template.bind({});
+GraphStory.storyName = 'Graph';
+GraphStory.args = {
+  yLabelStep: 50,
+  xLabelStep: 60,
+  labels,
+  data: [
+    labels.map((_) => {
+      y += 0.008;
+      return perlin.get(1, y) * 300 + 200;
+    }),
+  ],
+  previousData: Math.random() * 150,
+};
+
+export const GraphCloseEarlyStory = Template.bind({});
+
+GraphCloseEarlyStory.storyName = 'Graph Close Early';
+GraphCloseEarlyStory.args = {
+  yLabelStep: 50,
+  xLabelStep: 60,
+  labels,
+  data: [
+    labels.map((_, index) => {
+      y += 0.008;
+      return index < 300 ? perlin.get(1, y) * 300 + 200 : undefined;
+    }),
+  ],
+  previousData: Math.random() * 150,
+};
+
+export const GraphTwoPartsStory = Template.bind({});
+
+const dataWholeLine = labels.map((_) => {
+  y += 0.008;
+  return perlin.get(1, y) * 300 + 200;
+});
+
+GraphTwoPartsStory.storyName = 'Graph Two Parts';
+GraphTwoPartsStory.args = {
+  yLabelStep: 50,
+  xLabelStep: 60,
+  labels,
+  data: [
```

Comment:
I think it might be worth foregoing labels and data in the available controls.  It's a lot of data that I don't think people are likely to interact with since they're both so big.

Reply:
![Screen Shot 2022-11-15 at 11 08 32 AM](https://user-images.githubusercontent.com/106985561/201969691-c3f72581-73d9-4c5b-8bf9-8b02cb17e059.png)
![Screen Shot 2022-11-15 at 11 08 41 AM](https://user-images.githubusercontent.com/106985561/201969709-e5afed09-da7b-4526-99b8-fd89ed642fb0.png)